.circleci/config.yml

```diff
@@ -112,6 +112,7 @@ workflows:
           slack_channel: << pipeline.parameters.alerts-slack-channel >>
           context:
             - hmpps-common-vars
+          cache_key: "v2_0"
```

Comment:
This should fix https://mojdt.slack.com/archives/C01RFBVUQ4B/p1648786406694339 issue: _Unable to connect to the dependency-check database_